go/vt/vttablet/tabletserver/dt_executor.go

```diff
@@ -33,19 +33,21 @@ import (
 
 // DTExecutor is used for executing a distributed transactional request.
 type DTExecutor struct {
-	ctx      context.Context
-	logStats *tabletenv.LogStats
-	te       *TxEngine
-	qe       *QueryEngine
+	ctx       context.Context
+	logStats  *tabletenv.LogStats
+	te        *TxEngine
+	qe        *QueryEngine
+	shardFunc func() string
```

Comment:
mind adding a comment about what this is?

Reply:
I will add on a follow up PR.